x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -451,6 +462,14 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
     expireTime,
   ]);
 
+  const handleOnSubmit = useCallback(() => {
+    if (wildcardWarningExists) {
```

Comment:
I don't think it would make sense in this case since I needed to separate the original submit handler for the confirmation modal. We allows the user to add the exception without addressing the warning in the confirmation modal so I wanted to have that logic on its own for the confirmation modal submit handler.